scripts/qa_crowbarsetup.sh

```diff
@@ -5038,6 +5038,37 @@ function onadmin_runlist
     done
 }
 
+function onadmin_external_ceph
+{
+    external_ceph_nodes=$1
+    shift
+    local ceph_cluster=($@)
+    local node
+    local i=0
+    local done=0
+    local ses_repo="http://$adminip:8091/suse-$suseversion/$arch/repos/SUSE-Enterprise-Storage-$sesversion"
+    for node in ${ceph_cluster[@]}; do
+	set_node_alias $node ceph$i
+	let "i+=1"
+        ssh $node zypper ar -f ${ses_repo}-Pool/ SUSE-Enterprise-Storage-$sesversion-Pool
+        ssh $node zypper ar -f ${ses_repo}-Updates/ SUSE-Enterprise-Storage-$sesversion-Updates
+	for tries in $(seq 2); do
+	    ${SCRIPTS_DIR}/allocate_ip.sh $node storage host
+	    if [ $? -eq 0 ]; then
+	        done=1
+                break
+	    fi
+	done
+        if [ $done -eq 0 ]; then
+	    echo "Failed to assign a network to $node"
```

Comment:
please use spaces instead of tabs for indention.

Reply:
Thanks, alot of this debugging was done on a mkch* server, have now updated the vimrc configuration to correctly deal with tabs.